src/FacetManipulationTrait.php

```diff
@@ -65,26 +66,21 @@ protected function transformResultsToOptions(array $results): array {
    *   The label.
    */
   protected function getDefaultFilterValuesLabel(FacetInterface $facet, ListPresetFilter $filter): string {
-    // Back up the original active items so we can set them back after
-    // we used the facet to generate the display values from the filter.
-    $active_items = $facet->getActiveItems();
     $filter_values = $filter->getValues();
-    $facet->setActiveItems($filter_values);
-    $results = $this->processFacetResults($facet);
+
+    $clone = clone $facet;
+    $this->rebuildFacet($clone, $filter->getValues());
```

Comment:
The `rebuildFacet` is there so that in any moment, regardless of execution of the list, we can build a facet and fill it with processed results. This is critical for places like the default forms where we don;t have the context and we need to run things from scratch. So i call the query type on the facet to build and then process the facet to do any extra things that are needed. See the docs above the method as well.